codes/quantum/qubits/xzzx.yml

```diff
@@ -0,0 +1,31 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: xzzx
+physical: qubits
+logical: qubits
+
+name: 'XZZX surface code'
+introduced: '\cite{arXiv:2009.07851}'
+
+description: 'A family of stabilizer codes whose generators are \(XZXZ\) Pauli strings associated, clock-wise, to the vertices of each face of a two-dimensional lattice (with a qubit located at each vertex of the tessellation)'
+
+protection: 'As stabilizer \([[n=O(d^2), k=O(1), d]]\).'
+
+features:
+  threshold:
+  - '\(50\%\) with pure \(X, Y\) or \(Z\) errors.'
+  - '\(18.7\%\) at standard depolarising noise.'
+
+relations:
+  parents:
+    - code_id: stabilizer
+      detail: ''
+  cousins:
+    - code_id: surface
```

Comment:
The XZZX is just a special case of the surface code so this should be the parent.

Reply:
Due to the role of subsystem symmetries in the finite bias decoder, one can argue that fracton models should be a cousin.

Reply:
Please include surface as a parent and fracton as a cousin (I will create a dummy entry).

Reply:
Sorry, I'm not sure. Is the argument for finite bias or infinite?

Reply:
@mgullans I added a comment re subssytem symmetries, is there a ref?

Reply:
https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.2.013303